source/release-notes/1.1.txt

```diff
@@ -0,0 +1,44 @@
+.. _c2c-release-notes-1.1:
+
+===================================
+Release Notes for ``mongosync`` 1.1
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.1 Released January 13, 2023
+
+
+Filtered Sync
+-------------
+
+{+c2c-product-name+} provides continuous data synchronization or a 
+one-time data migration between two MongoDB clusters. You can use
+filtered sync to control the data that the :ref:`mongosync
+<c2c-mongosync>` utility transfers between the source and destination
+clusters.
```

Comment:
can we be more prescriptive that we allow users to specify "databases" and "collections" that they want to sync?

Reply:
I can imagine if later if we add more capability to filtered sync (e.g. query level), we may mention it in future release notes